.github/workflows/integration-test.yml

```diff
@@ -171,10 +171,13 @@ jobs:
           # Execute test by linkis-cli
           POD_NAME=`kubectl get pods -n linkis -l app.kubernetes.io/instance=linkis-demo-mg-gateway -o jsonpath='{.items[0].metadata.name}'`
           kubectl exec -it -n linkis  ${POD_NAME} -- bash -c " \
-          sh /opt/linkis/bin/linkis-cli -engineType shell-1 -codeType shell -code \"pwd\" ";
-
-          kubectl exec -it -n linkis  ${POD_NAME} -- bash -c " \
-          sh /opt/linkis/bin/linkis-cli -engineType python-python2 -codeType python -code   'print(\"hello\")' "
+          sh /opt/linkis/bin/linkis-cli --async true -engineType shell-1 -codeType shell -code \"pwd\" ";
```

Comment:
Asynchronous cannot be used and the task needs to be successful.

Reply:
I have rolled back this part of the code.